routes/FeedbackListController.php

```diff
@@ -25,6 +25,7 @@ public static function get(): :xhp {
       # Append the applicant to the table as a new row
       $table_body->appendChild(
         <tr class={$feedback->getComments() != '' ? "success" : ""}>
```

Comment:
So the #id trick only works if you set the id property on the element. So instead of adding a new data column to the row, add id={$row['id']} as property of the row. That'll make the page jump to this location as desired.